Automattic/jetpack
# Add/one click sharing section

Adds the previously added Quck-share panel to the editor too

## Proposed changes:

* Updates share-buttons a bit to match designs more
* Adds the new Dropdown to our panel

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Open up a new post, there should not be anything new.
* After publishing you should see the new Quick share Button if our panel is open
* Clicking the icon should open the Quick share dropdown
* Clicking on any of the icons there should work
* Clicking the learn more on the dropdown should open the help modal and close the dropdown.
* Test on mobile
